### PR TITLE
Added method get_other_companies_founded_by_person_with_the_same_full…

### PR DESCRIPTION
**Pull request by IegorSoboliev**
…name to PepSerializer

### CODE REVIEW

File: business_register/serializers/pep_serializers.py [hunk 1]
```diff
@@ -25,11 +26,22 @@ class PepSerializer(serializers.ModelSerializer):
     related_persons = RelatedPersonSerializer(many=True)
     # related_companies = CompanyLinkWithPepSerializer(many=True)
     related_companies = serializers.SerializerMethodField()
+    other_companies_founded_by_person_with_the_same_fullname = serializers.SerializerMethodField()
```
Comment (roman-tiukh) [inline business_register/serializers/pep_serializers.py:29]: this is the longest property name i have ever seen =))), maybe shorten it?

File: business_register/serializers/pep_serializers.py [hunk 2]
```diff
@@ -25,11 +26,22 @@ class PepSerializer(serializers.ModelSerializer):
...
 
     def get_related_companies(self, obj):
         qs = obj.related_companies.select_related('company', 'company__company_type', 'company__status').all()
         return CompanyLinkWithPepSerializer(qs, many=True).data
 
+    def get_other_companies_founded_by_person_with_the_same_fullname(self, pep):
+        founder_of = Founder.objects.filter(name__contains=pep.fullname)
+        other_companies_founded_by_person_with_the_same_fullname = []
+        for founder in founder_of:
+            if founder.company not in pep.related_companies:
```
Comment (roman-tiukh) [inline business_register/serializers/pep_serializers.py:39]:
```suggestion
        founder_of = Founder.objects.filter(
            name__contains=pep.fullname
        ).exclude(company__in=pep.related_companies)
        other_companies_founded_by_person_with_the_same_fullname = []
        for founder in founder_of:
```

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: business_register/serializers/pep_serializers.py (hunk 1 above)
Comment (IegorSoboliev) [reply business_register/serializers/pep_serializers.py:30]: My first language was Java). Done

File: business_register/serializers/pep_serializers.py (hunk 2 above)
Comment (IegorSoboliev) [reply business_register/serializers/pep_serializers.py]: Thought about this

Comment (IegorSoboliev) [reply business_register/serializers/pep_serializers.py]: But used another logic in the end. Please, review the new one

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**